sample/MozillaAllHands_17/tracks

```diff
@@ -0,0 +1,230 @@
+[ 
+    {
+        "id": 455, 
+        "name": "All Hands", 
+        "color": "#2E7D32",  
```

Comment:
Please add font_color too as suggested by @Princu7 It will make for a better web app.